patchwork-events-rendering/src/main/java/net/minecraftforge/client/event/TextureStitchEvent.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Minecraft Forge, Patchwork Project
+ * Copyright (c) 2016-2020, 2019-2020
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.client.event;
+
+import java.util.Set;
+
+import net.minecraftforge.eventbus.api.Event;
+
+import net.minecraft.client.texture.SpriteAtlasTexture;
+import net.minecraft.util.Identifier;
+
+public class TextureStitchEvent extends Event {
+	private final SpriteAtlasTexture map;
+
+	public TextureStitchEvent(SpriteAtlasTexture map) {
+		this.map = map;
+	}
+
+	public SpriteAtlasTexture getMap() {
+		return map;
+	}
+
+	/**
+	 * Fired when the TextureMap is told to refresh it's stitched texture.
+	 * Called before the {@link net.minecraft.client.texture.SpriteAtlasTexture} are loaded.
```

Comment:
No need to fully qualify, it's already imported
```suggestion
	 * Called before the {@link SpriteAtlasTexture}s are loaded.
```